dev/docker/docker-compose-register.yml

```diff
@@ -1,9 +1,9 @@
-x-xmtpd-cli-image: &xmtpd-cli-image ghcr.io/xmtp/xmtpd-cli:sha-d4e697c
-
 services:
   register-node-1:
     platform: linux/amd64
-    image: *xmtpd-cli-image
+    build:
+      dockerfile: ./dev/docker/Dockerfile-cli
+      context: ../../
```

Comment:
Are we building it every time?

Reply:
only if the go code changes. That is why I need #821 . Otherwise it gets annoying